Makefile

```diff
@@ -8,3 +8,12 @@ impala-test:
 
 clean-pyc:
 	find . -name "*.pyc" -exec rm -rf {} \;
+
+develop: clean-pyc
+	python setup.py develop
+
+lint:
+	flake8
+
+test:
+	pytest --pyargs ibis
```

Comment:
this should probably include `-m 'not impala and not hdfs'` because you can have all the python deps necessary to run impala queries but no impala nodes up to test against.